rdrf/rdrf/db.py

```diff
@@ -0,0 +1,78 @@
+from io import StringIO
+import os
+
+from django.core.management import call_command
+from django.db import connections
+
+class RegistryRouter:
```

Comment:
Sharing the DB is something I want to keep because it's useful for doing quick tests, etc.

But I added a system check deployment error to fail when the clinical database hasn't been configured. 014f4aa71326e28303167799d12146873820ee4d